themes/ddbasic/sass/layout/_layout.scss

```diff
@@ -573,10 +573,6 @@ body {
   text-align: center;
 }
 
-.primary-content table img {
```

Comment:
This was implemented in https://inlead.atlassian.net/browse/DDBEASY-711 in order for a page like this to work: https://www.vardebib.dk/page/personale. Please make sure that this is still working after the removal of this code.